values.yaml

```diff
@@ -357,13 +380,40 @@ podAnnotations: {}
 ###################################
 
 # Defines privilege and access control settings for the node pods.
-podSecurityContext: {}
+podSecurityContext:
+  runAsNonRoot: true
+  runAsUser: 10001
+  runAsGroup: 10001
+  fsGroup: 10001
+  fsGroupChangePolicy: OnRootMismatch
+  seccompProfile:
+    type: RuntimeDefault
 
 # Defines privilege and access control settings for the node container running GraphDB.
-securityContext: {}
+securityContext:
+  allowPrivilegeEscalation: false
+  readOnlyRootFilesystem: true
+  capabilities:
+    drop: [ "ALL" ]
+  seccompProfile:
```

Comment:
I put `seccompProfile` in both the pod and the container contexts cause some tools were complaining. TBH, I think that having the profile in `podSecurityContext` is enough.

Reply:
Lets leave them both for now.